### PR TITLE
P0:Adding  alert badge button

### PR DESCRIPTION
**Pull request by eldhoabraham94**
Adding Alert, Button, Badge
Adding fontawsome for icons

### CODE REVIEW

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.btn{
+    border-radius: 0.5rem;
```
Comment (deekshithmd) [inline components/button/button.css]: giving border radius in px will be good

**Review by deekshithmd — APPROVED** [review]

File: components/common.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+:root{
+    /* FONT SIZE  */
+
+    font-size:62.5%;
+
+    /* COLORS */
+
+    --ALERT-INFO-TEXT:#004085;
```
Comment (dev-ashishsethi) [inline components/common.css:19]: naming is good

**Review by dev-ashishsethi — APPROVED** [review]
looks good

File: components/button/button.css (hunk 1 above)
Comment (eldhoabraham94) [reply components/button/button.css]: @deekshithmd  Updated. Thank you

**Pull request merged**